modules/statistics/php/stats_behavioural.class.inc

```diff
@@ -63,18 +63,23 @@ class Stats_Behavioural extends \NDB_Form
     {
         parent::setup();
 
-        $DB =& \Database::singleton();
+        $DB   =& \Database::singleton();
```

Comment:
```suggestion
        $factory = NDB_Factory::singleton();
        $DB      = $factory->database();
```
New way to create DB object to facilitate unit testing.
Not a blocker but would be nice.

Reply:
should be :

```suggestion
        $factory = \NDB_Factory::singleton();
```

with the \ in front of NDB